config/webhook.yaml

```diff
@@ -140,6 +140,10 @@ spec:
           initialDelaySeconds: 5
           periodSeconds: 10
           timeoutSeconds: 5
+      securityContext:
+        seLinuxOptions:
```

Comment:
I'm actually wondering if we need seLinuxOptions here since it's basically redundant with the container security context?

Reply:
Looking at the `seLinux` setup from https://kubernetes.io/docs/reference/access-authn-authz/psp-to-pod-security-standards/ it seems that with the restricted label, it specified the podTemplate to be `seLinux: mustRunAs` with `user` as undefined. I was wondering if this essentially mean it could be 0 as root?

Reply:
Yeah typically root uid/gid is 0.

If you want a simple test to add to the e2e test try to add a simple a pod that runs as root to ensure that PSS is correctly blocking the pod.